build.gradle

```diff
@@ -3,9 +3,13 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath "io.realm:realm-gradle-plugin:2.3.0-SNAPSHOT"
```

Comment:
😟 We shouldn't be using snapshots. As soon as 2.3.0 hits stable, we should take care of this.